addons/binding/org.openhab.binding.facerecognition/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="facerecognition"
+						xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+						xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+						xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="facerecognition">
+		<label>Face recognition</label>
+		<description>User authentication using face recognition</description>
+		<channel-groups>
+			<channel-group typeId="live-group" id="live"></channel-group>
+			<channel-group typeId="accessGranted-group" id="accessGranted"></channel-group>
+			<channel-group typeId="accessDenied-group" id="accessDenied"></channel-group>
+		</channel-groups>
+		<config-description>
+			<parameter name="camera" type="text">
+				<required>true</required>
+				<context>network_address</context>
+				<description>URL for receiving camera snapshots</description>
+				<default>http://IP/image.jpg</default>
```

Comment:
Better not do that. The default value will satisfy the "required" attribute, but the default is not a valid address.